diff.py

```diff
@@ -455,6 +480,7 @@ def create_project_settings(settings: Dict[str, Any]) -> ProjectSettings:
         ),
         objdump_executable=get_objdump_executable(settings.get("objdump_executable")),
         objdump_flags=settings.get("objdump_flags", []),
+        expected_directory=settings.get("expected_directory", "expected/"),
```

Comment:
can you add this commented-out to diff_settings.py? and maybe rename it "expected_dir" for consistency with "build_dir"?